src/components/common/SearchPanels/SearchPanelTravelTips.jsx

```diff
@@ -0,0 +1,24 @@
+import SearchIcon from '../../../assets/img/searchIcon.svg';
+
+const SearchPanelTravelTips = () => {
+  return (
+    <form className="flex h-10 w-[23rem] rounded-10px bg-light-color pl-4 drop-shadow-lg">
```

Comment:
Don't forget to add `event.preventDefault()` to prevent form submission and page reload

Reply:
Thought, if my component without any logic - I don't need to add any handles. Thought. when I add functionality to search, I will write all this logic.